spec/javascripts/report-a-problem-form-spec.js

```diff
@@ -30,9 +30,13 @@ describe("form submission for reporting a problem", function () {
     it("should submit using ajax to the action specified by the form", function() {
       var args;
       spyOn($, "ajax");
+      spyOn(GOVUK.analytics, "trackEvent");
       $form.triggerHandler('submit');
 
       expect($.ajax).toHaveBeenCalled();
+      expect(GOVUK.analytics.trackEvent).toHaveBeenCalledWith(
```

Comment:
Yes, sorry if my comment was not clear enough. I would do the following change:

* From:

```javascript
it(“should submit using ajax to the action specified by the form”) {
// Test the form is being sent
// Test the event is sent
}
```

* To:

```javascript
it(“should submit using ajax to the action specified by the form”) {
// Test the form is being sent
}

it(“it should track the event: GOVUK Send Feedback”) {
// Test the event is sent
}
```

This way, when you run tests individually, you can read what the actual behaviour of the function / Module / Object is. If you decide, for example, to stop sending those events, you just need to remove the unit test, and not updating an existing one.